src/cdk-experimental/testing/tests/harnesses/sub-component-harness.ts

```diff
@@ -6,12 +6,21 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {ComponentHarness} from '../../component-harness';
+import {ComponentHarness, HarnessPredicate} from '../../component-harness';
 import {TestElement} from '../../test-element';
 
 export class SubComponentHarness extends ComponentHarness {
   static readonly hostSelector = 'test-sub';
 
+  static with(options: {title?: string | RegExp, itemCount?: number} = {}) {
+    return new HarnessPredicate(SubComponentHarness)
+        .addOption('title', options.title,
+            async (harness, title) =>
+                HarnessPredicate.stringMatches(await (await harness.title()).text(), title))
```

Comment:
Having to do two awaits in a row feels a little weird. Since more or less everything is async, can we get away with having `stringMatches` accept a promise?

Reply:
That seems reasonable